dev/bazel/deps/mkl.bzl

```diff
@@ -22,8 +22,10 @@ mkl_repo = repos.prebuilt_libs_repo_rule(
     ],
     libs = [
         "lib/libmkl_core.a",
-        "lib/libmkl_sequential.a",
         "lib/libmkl_intel_ilp64.a",
+        "lib/libmkl_sequential.a",
+        "lib/libmkl_tbb_thread.a",
```

Comment:
Is this required?

Reply:
yes, previously we had 2 different mkl deps in bazel, and it was split as seq and thread versions. Now we agreed that we will use thread mkl in all scenarios, so, I replaced "lib/libmkl_sequential.a", with "lib/libmkl_tbb_thread.a",